### PR TITLE
fzf: update to 0.29.0

### PR DESCRIPTION
**Pull request by Jlll1**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by paper42** [comment]
please squash the commits, then this PR should be ready

**Comment by Jlll1** [comment]
Done! Thank you!

File: srcpkgs/fzf/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'fzf'
 pkgname=fzf
-version=0.28.0
-revision=2
+version=0.29.0
+revision=1
 build_style=go
 go_import_path="github.com/junegunn/fzf"
 hostmakedepends="git"
```
[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/fzf/template:7]: this was fixed, but then the fix was reverted

**Comment by leahneukirchen** [comment]
104fe2e7da4

**Pull request closed without merging**